helm/cluster-vsphere/Chart.yaml

```
@@ -0,0 +1,18 @@
+annotations:
+  application.giantswarm.io/team: "rocket"
```

```suggestion
  application.giantswarm.io/team: rocket
```
Doesn't need to be quoted.